doc/source/whatsnew/v0.24.0.txt

```diff
@@ -295,6 +295,8 @@ Datetimelike
 ^^^^^^^^^^^^
 
 - Fixed bug where two :class:`DateOffset` objects with different ``normalize`` attributes could evaluate as equal (:issue:`21404`)
+- Bug in :class:`Index` with ``datetime64[ns, tz]`` dtype that did not localize integer data correctly (:issue:`20964`)
+- Fixed bug where :meth:`DataFrame.describe` and :meth:`Series.describe` on tz-aware datetimes did not show `first` and `last` result (:issue:`21328`)
```

Comment:
One more nit: Could you move this entry to the timezones section below?

Also the entry above this one exists in the timezone section already i.e. you can remove it.

Thanks for you patience.

Reply:
Done, no problem.